libbeat/docs/tab-widgets/ingest-data.asciidoc

```diff
@@ -0,0 +1,68 @@
+// tag::modules[]
+[float]
+[[enable-modules]]
+===== Enable and configure data collection modules
+
+. Identify the modules you need to enable. To see a list of available
+<<filebeat-modules,modules>>, run:
++
+--
+include::{libbeat-dir}/tab-widgets/list-modules-widget.asciidoc[]
+--
++
+Can't find a module for your file type? Skip this section and
+<<configuration-filebeat-options,configure the input>> manually.
+
+. From the installation directory, enable one or more modules. For example, the
+following command enables the `system`, `nginx`, and `mysql` module
+configs:
++
+--
+include::{libbeat-dir}/tab-widgets/enable-modules-widget.asciidoc[]
+--
+
+. In the module configs under `modules.d`, change the module settings to match
+your environment.
++
+For example, log locations are set based on the OS. If your logs aren't in
+default locations, set the `paths` variable:
++
+--
+[source,yaml]
+----
+- module: nginx
+  access:
+    var.paths: ["/var/log/nginx/access.log*"] <1>
+----
+--
+
+To see the full list of variables for a module, see the documentation under
+<<filebeat-modules>>.
+
+include::{libbeat-dir}/shared/config-check.asciidoc[]
+
+// end::modules[]
+
+// tag::application-logs[]
+[float]
+[[setup-application-logs]]
+===== Ingest application logs
+
+This section is about best practices for ingesting logs from your own applications,
```

Comment:
I think this section is a little too hand wavy. As a user, I want to get started quickly. Tell me exactly what I need to do (preferably in steps). Because what you do depends a lot on what the logs look like, it might help here to provide steps with concrete examples.